portal/static/js/admin.js

```diff
@@ -421,7 +421,51 @@ AdminTool.prototype.handleDownloadModal = function() {
     $("input[name='instrument'], input[name='downloadType']").on("click", function() {
         if ($(this).is(":checked")) $("#_downloadMessage").text("");
     });
-}
+};
+
+AdminTool.prototype.getReportModal = function(patientId) {
+  $("#patientReportModal").modal("show");
+  $("#patientReportLoader").removeClass("tnth-hide");
+  tnthAjax.patientReport(patientId, function(data) {
+      if (data) {
+        if (!data.error) {
+            if (data["user_documents"] && data["user_documents"].length > 0) {
+              var existingItems = {};
+              /*
+               * sort to get the latest first
+               */
+              var documents = data["user_documents"].sort(function(a,b){
+                 return new Date(b.uploaded_at) - new Date(a.uploaded_at);
+              });
+              var content = "<table class='table-bordered table-condensed table-responsive tnth-table'>";
+              content += "<TH>"+ i18next.t("Report Name") + "</TH><TH>" + i18next.t("Generated (GMT)") + "</TH><TH>" + i18next.t("Downloaded") + "</TH>";
+              documents.forEach(function(item) {
+                  /*
+                   * only draw the most recent, same report won't be displayed
+                   */
+                  if (!existingItems[item["contributor"]]) {
+                    content += "<tr>" +
+                              "<td>" + item["filename"] + "</td>" +
+                              "<td>" + tnthDates.formatDateString(item["uploaded_at"], "iso") + "</td>" +
+                              "<td class='text-center'>" + '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"])+ '"><i class="fa fa-download"></i></a>' + "</td>" 
```

Comment:
Found the one string - "Download"! ;)

Reply:
@ivan-c Ivan, thank you! I added the tag and added another column to the report table.  Let me know if you see any more issue.